lib/src/widgets/calendar_page.dart

```diff
@@ -6,31 +6,62 @@ import 'package:flutter/widgets.dart';
 class CalendarPage extends StatelessWidget {
   final Widget Function(BuildContext context, DateTime day)? dowBuilder;
   final Widget Function(BuildContext context, DateTime day) dayBuilder;
+  final Widget Function(BuildContext context, DateTime day)? weekNumberBuilder;
   final List<DateTime> visibleDays;
   final Decoration? dowDecoration;
   final Decoration? rowDecoration;
   final bool dowVisible;
+  final bool weekNumberVisible;
+  final double? dowHeight;
 
   const CalendarPage({
     Key? key,
     required this.visibleDays,
     this.dowBuilder,
     required this.dayBuilder,
+    this.weekNumberBuilder,
     this.dowDecoration,
     this.rowDecoration,
     this.dowVisible = true,
+    this.weekNumberVisible = false,
+    this.dowHeight,
   })  : assert(!dowVisible || dowBuilder != null),
```

Comment:
Please add a similar assertion for `weekNumberVisible` and `weekNumberBuilder`. Also, please update the existing assertion to include `dowHeight`, just like in CalendarCore widget.